backend/packages/wps-shared/src/wps_shared/geospatial/cog.py

```diff
@@ -0,0 +1,81 @@
+"""
+Cloud-Optimized GeoTIFF (COG) generation utilities.
+
+This module provides functions for creating web-optimized COGs from geospatial rasters,
+including reprojection to web-friendly coordinate systems like EPSG:3857.
+"""
+
+import logging
+
+from osgeo import gdal
+
+from wps_shared.geospatial.geospatial import GDALResamplingMethod
+
+logger = logging.getLogger(__name__)
+
+
+def generate_web_optimized_cog(
+    input_path: str,
+    output_path: str,
+    target_srs: str = "EPSG:3857",
```

Comment:
`EPSG:3857` seems to be hard coded in a couple/few places. What do you think about creating a constant for it in `geospatial.py`?